maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/service/RepOrderService.java

```diff
@@ -51,6 +54,17 @@ public RepOrderDTO updateDateCompleted(final UpdateAppDateCompleted updateAppDat
                 .updateAppDateCompleted(updateAppDateCompleted.getRepId(), updateAppDateCompleted.getAssessmentDateCompleted()));
     }
 
+    @Transactional
+    public RepOrderDTO create(final CreateRepOrder createRepOrder) {
+        log.info("create rep order - Transaction Processing - Start");
+        LocalDate dateNow = LocalDate.now();
+        RepOrderEntity repOrderEntity = new RepOrderEntity();
+        repOrderEntity.setId(createRepOrder.getRepId());
+        repOrderEntity.setDateCreated(dateNow);
+        repOrderMapper.createRepOrderToRepOrderEntity(createRepOrder, repOrderEntity);
+        return repOrderMapper.repOrderEntityToRepOrderDTO(repOrderImpl.createRepOrder(repOrderEntity));
```

Comment:
The repOrderImpl.createRepOrder(repOrderEntity) is actually doing the saving of the entity so my recommendation is to put it on its own line rather than passed as a param here.

The reason for this is if there are any errors thrown during saving the entity it'll show the correct line in the stacktrace (it might confusingly show this repOrderMapper line).

Reply:
Makes sense, done.